Wellthier/Cells/PostCell.m

```diff
@@ -20,7 +20,12 @@ - (void)setPostDetails: (Post *)post {
     if (post[@"image"]) {
         self.postImageView.file = post[@"image"];
         [self.postImageView loadInBackground];
+        self.postImageView.hidden = NO;
+    } else {
+        self.postImageView.image = nil;
```

Comment:
You don't need this line.

Reply:
was getting glitchy behavior without the else statement where pictures from a separate post would be displayed in the cell of a post without a picture